compiler/testData/codegen/bytecodeText/conditions/nullCompareConst.kt

```diff
@@ -0,0 +1,11 @@
+// IGNORE_BACKEND: JVM
```

Comment:
Please note that you can customize bytecode tests by backend via 
..common_templates...
// JVM_TEMPLATES
...
// JVM_IR_TEMPLATES
...
, see for example forInReversedArrayIndices.kt

Reply:
So IGNORE_BACKEND could be removed

Reply:
Ah, neat! And that is preferred over the individual tests?

Reply:
Yes, to avoid test data duplication and decrease number of IGNORE directives